feat: added the latest Sonnet 3.5 and Haiku 3.5

milutinke opened the pull request:
Should we remove the `claude-3-5-sonnet-20240620` (Claude Sonnet 3.5 (old)) since there is no pricing difference and the new model is slightly better?

syndicate604 commented:
No as its 4x cheaper some people may want to use it, but right now unless your on Tier 2 Anthropic your not getting far on bolt anyways

milutinke commented:
> No as its 4x cheaper some people may want to use it, but right now unless your on Tier 2 Anthropic your not getting far on bolt anyways

I don't think there is a price difference between the 3.5 (old) and the 3.5 (new)

wonderwhy-er commented:
I just checked out, merged master, fixed conflict with constants.ts and tested with API key
Worked fine, I think if you rebase we can merge

milutinke commented:
> I just checked out, merged master, fixed conflict with constants.ts and tested with API key Worked fine, I think if you rebase we can merge

Fixed the conflict, checked out the `git checkout c35211f4a075763eaa8da54baa362900be1d61fb` and tested, works.
